client/src/explorer_common/explorer_components.js

```diff
@@ -49,7 +49,6 @@ export const ExplorerHeader = ({
             }
             onClick={() => col_click(id)}
             tabIndex={0}
-            role="button"
```

Comment:
Basically not great practice to have buttons nested within buttons as it can cause problems with screen readers (https://dequeuniversity.com/rules/axe/4.2/nested-interactive). By removing the `role="button"` the header of this table is still clickable and will toggle the sorting but will no longer have the "hand click" mouse icon that you see when you hover other clickable elements. I feel like this comes back to the original issue with these toggling arrows that don't actually have the functionality within them?

Reply:
Original issue?

For the cursor: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

Reply:
Original issue that had been brought up when Christina was working on something in the report builder (maybe storybooking those arrows?). Rather than the individual arrows being a "proper" button, they act as a toggle for each other. For example: if you click on the left upwards arrow, it won't activate that specific filter but it will toggle the filter (up or down). So you could keep clicking on the left upwards arrow and keep switching back and forth between filtering upwards or downwards. Especially confusing when starting off with no filter and then clicking on the left upwards arrow but the right downwards one activates.

Reply:
Ah, makes sense. Don't see any open issues for that, make one if you want.